docs/documentation/stories.md

```diff
@@ -29,3 +29,4 @@
  - [Deploy to GitHub Pages](stories/github-pages)
  - [Linked Library](stories/linked-library)
  - [Multiple apps](stories/multiple-apps)
+ - [Automated testing](stories/automated-testing)
```

Comment:
Could we name it `Continuous Integration`? Someone looking for travis instructions is more likely to find CI than "Automated Testing".